janus_server/Cargo.toml

```diff
@@ -59,7 +59,7 @@ testcontainers = { version = "0.14.0", optional = true }
 thiserror = "1.0"
 tokio = { version = "^1.20", features = ["full", "tracing"] }
 tokio-postgres = { version = "0.7.7", features = ["with-chrono-0_4", "with-serde_json-1", "with-uuid-1", "array-impls"] }
-tonic = { version = "0.8.0", optional = true, features = ["tls", "tls-webpki-roots"] }
+tonic = { version = "0.6.2", optional = true, features = ["tls", "tls-webpki-roots"] }  # keep this version in sync with what opentelemetry-otlp uses
```

Comment:
Consider also adding a comment to the line for `opentelemetry-otlp`, as someone clicking through a dependabot PR to update that package may not notice this comment.

Reply:
I wonder if dependabot will preserve the trailing comment if/when it bumps either version. Maybe it's best to put the comment above the dependencies, just in case?

Reply:
I checked [the source code](https://github.com/dependabot/dependabot-core/blob/main/cargo/lib/dependabot/cargo/file_updater/manifest_updater.rb), and it does a series of regex replacements, so our comments should be fine.